main.go

```diff
@@ -1,17 +1,22 @@
 package main
 
 import (
+	"fmt"
 	"io"
 	"log"
 	"os"
 	"path"
 
+	flag "github.com/spf13/pflag"
 	"github.com/surface-security/scanner-go-entrypoint/scanner"
 )
 
+const DEFAULT_USER_AGENT = "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_14_3) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/75.0.3770.142 Safari/537.36 github.com/surface-security/scanner-httpx"
+
 func main() {
 	s := scanner.Scanner{Name: "httpx"}
 	options := s.BuildOptions()
+	userAgent := flag.String("ua", DEFAULT_USER_AGENT, "Choose user-agent - use empty value for a random UA")
```

Comment:
Since this is using a default value, there will not be a random value ever, or an empty `User-Agent` flag so that the `httpx` scanner sets a random value, right?
 
```suggestion
	userAgent := flag.String("ua", DEFAULT_USER_AGENT, "Override user-agent value - defaults to one provided")
```

Reply:
No

```
--ua ""
```

Hence the help text saying "set to empty".

It might not really be intuitive though but I wanted the default UA to be "tagged".

We can change that logic and add new flag for "random agent" (instead of using the empty condition) but it might get trickier because users might use extra flags that conflict with it and end up misled